praw/models/helpers.py

```diff
@@ -23,6 +23,35 @@ def __call__(self, id):  # pylint: disable=invalid-name,redefined-builtin
         """
         return LiveThread(self._reddit, id=id)
 
+    def info(self, ids):
+        """Fetch information about each live thread in ``ids``.
+
+        :param ids: A list of IDs for live threads.
+        :returns: A generator that yields :class:`.LiveThread` instances
```

Comment:
This returns `praw.models.listing.Listing` instance, so the doc is incorrect. I'll fix that.

I don't know this endpoint's limitation. [/api/info have 100 item limit](https://github.com/reddit/reddit/blob/87c315b8aefa16f4ae8e62d8b64699582e4561e4/r2/r2/controllers/api.py#L181), but [I can't find similar limit for /api/live/by_id](https://github.com/reddit/reddit-plugin-liveupdate/blob/80ec0d53759c3e33030e8c4bc417e019cd5c825c/reddit_liveupdate/controllers.py#L1158). I can't test actual limit because live thread listing is not available (it's admin only).